src/machine/usb/usb.go

```diff
@@ -75,9 +80,11 @@ const (
 	CDC_ENDPOINT_ACM  = 1
 	CDC_ENDPOINT_OUT  = 2
 	CDC_ENDPOINT_IN   = 3
-	HID_ENDPOINT_IN   = 4
-	MIDI_ENDPOINT_OUT = 5
-	MIDI_ENDPOINT_IN  = 6
+	HID_ENDPOINT_IN   = 4 // for Interrupt In
+	HID_ENDPOINT_OUT  = 5 // for Interrupt Out
+	MIDI_ENDPOINT_IN  = 6 // for Bulk In
+	MIDI_ENDPOINT_OUT = 7 // for Bulk Out
```

Comment:
This change affects more than just the rp2040.
However, the change is still a good one.

Reply:
> do you mean that the current MIDI implementation conflicts with this PR?

@deadprogram 
This change affects other microcontrollers.
The descriptor could also be changed, so it should work, but for some reason it does not work at the moment.
I will now look into this issue.